ctms/crud.py

```diff
@@ -929,3 +929,19 @@ def delete_acoustic_newsletters_mapping(dbsession, source):
     dbsession.delete(row)
     dbsession.commit()
     return row
+
+
+def gdpr_delete(db: Session, email_id: UUID4):
+    models = [
+        PendingAcousticRecord,
+        AmoAccount,
+        MozillaFoundationContact,
+        Newsletter,
+        Waitlist,
+        FirefoxAccount,
+        Email,
+    ]
```

Comment:
I wonder if we should add `ondelete='CASCADE'` to our `db.ForeignKey()` relatioships instead of doing it manually here at the Python level.

Ideally, deleting a `Email` record, should delete everything related to it, no?

Reply:
right, this could be handled through the database model :thinking: